#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_gcc43_build_fix.dpatch by Brian Nelson <pyro@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix for GCC 4.3 builds as reported in #452310

@DPATCH@
diff -urNad aspell-0.60.6~/common/string_map.hpp aspell-0.60.6/common/string_map.hpp
--- aspell-0.60.6~/common/string_map.hpp	2004-08-04 03:52:31.000000000 -0400
+++ aspell-0.60.6/common/string_map.hpp	2008-06-24 14:36:26.000000000 -0400
@@ -11,7 +11,7 @@
 #include "parm_string.hpp"
 #include "posib_err.hpp"
 #include "string_pair.hpp"
-#include "hash.hpp"
+#include "hash-t.hpp"
 #include "objstack.hpp"
 
 
diff -urNad aspell-0.60.6~/modules/speller/default/affix.hpp aspell-0.60.6/modules/speller/default/affix.hpp
--- aspell-0.60.6~/modules/speller/default/affix.hpp	2008-06-22 23:56:43.000000000 -0400
+++ aspell-0.60.6/modules/speller/default/affix.hpp	2008-06-24 14:36:26.000000000 -0400
@@ -109,7 +109,7 @@
     }
     WordAff * expand_suffix(ParmString word, const unsigned char * aff,
                             ObjStack &, int limit = INT_MAX,
-                            unsigned char * new_aff = 0, WordAff * * * l = 0,
+                            unsigned char * new_aff_2 = 0, WordAff * * * l = 0,
                             ParmString orig_word = 0) const;
     
   private:
